tests/testCompoundInstanceCatalog.py

```diff
@@ -288,6 +288,60 @@ def testCompoundCatalog(self):
         if os.path.exists(fileName):
             os.unlink(fileName)
 
+    def testSharedVariables(self):
```

Comment:
Really, I added it to make the `phoSimHeaderMap` API work with `CompoundInstanceCatalog`s.  I wanted to be able to set the `phoSimHeaderMap` on the `CompoundInstanceCatalog` and have that assignment propagate down to the individual `InstanceCatalog`s being iterated over. The fact that it can be used for `override_transformations`, `column_outputs`, etc. is just a happy accident.